### ISSUE TITLE
Tagging fails when updating quickFilters

### ISSUE THREAD

**Issue by RealRaven2000** [RealRaven2000/quickFilters#72]
It seems that every time a new quickFilters update is installed, it breaks the Tagging feature within Thunderbird until you do a Thunderbird restart.

Not a huge issue, but it is a bit frustrating to all of a sudden loose tagging and then need to reload Thunderbird to get it working again.

**Comment by RealRaven2000** [comment]
My guess is that this is connected to the tag listener which can be activated here:

![image](https://user-images.githubusercontent.com/6314674/130318465-07aadbad-cdf1-4f10-9822-394bf1a90803.png)

It is probably not removed correctly by the previous instance of quickFilters when it is stopped to install the new version. Note: when you uncheck the option it is likely this isn't removed either (I need to check that), only after disabling quickFilters or restarting Thunderbird, so this is the first thing I would like to implement.

**Comment by cpaschen** [comment]
Yes, this happens only in the session in which quickFilters is updated. After a restart of Thunderbird all issues are resolved then (until the next quickFilters update). It's been happening for at least a couple/few quickFilters updates/versions (I just kept forgetting to report it).
(And sorry, I didn't think to look at the console when it happened :-( )

I'm not presently using the tag changes for any filtering actions, so I've un-checked that box.
I'll try to watch when the next update comes out to see if it still causes this tag error and report back here.

**Comment by RealRaven2000** [comment]
I have inadvertently created a new issue #90 - had forgotten it already exists here. I uploaded a fix there, closing this one instead

**Comment by RealRaven2000** [comment]
Fixed in 5.4 released today
